CHANGELOG.md

```diff
@@ -21,5 +25,6 @@
 
 * Extract performance cops from rubocop-hq/rubocop repository. ([@composerinteralia][], [@koic][])
 
+[@bquorning]: https://github.com/bquorning
```

Comment:
Please move this link to the bottom of the changelog.

> If this is your first contribution to RuboCop project, add a link definition for the implicit link to the bottom of the changelog as [@username]: https://github.com/username.

https://github.com/rubocop-hq/rubocop/blob/master/CONTRIBUTING.md#changelog-entry-format

Reply:
I hadn’t seen that rule, and wondered if the list was alphabetical or in date order. 😄 

:+1: